tests/includes/Test_i18n.php

```
@@ -1003,4 +1005,374 @@ function () {
+			"<?php\nreturn array(\n\t'Hello' => 'Hallo',\n\t'Goodbye' => 'Tschüss',\n);\n"
+		);
+
+		// Set the version transient so no HTTP download occurs.
+		set_transient( 'wcpos_i18n_woocommerce-pos_de_DE', '1.8.7', WEEK_IN_SECONDS );
+
+		$this->http_responder = function () {
+			$this->fail( 'No HTTP request should be made when translation file is cached' );
+			return false;
+		};
+
+		$i18n = new i18n( 'woocommerce-pos', '1.8.7', $this->temp_lang_dir );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$i18n'. (undefined)

(UnusedLocalVariable)